aws/README.md

```diff
@@ -0,0 +1,13 @@
+# AWS
```

Comment:
Bucket name: scytale-artifacts
IAM sign-in: https://scytaleio.signin.aws.amazon.com/console
Access: @drrt @sigtrap 

Unsure what relies on the S3 URL